services/worker-manager/src/data.js

```diff
@@ -134,56 +133,157 @@ class WorkerPool {
   }
 }
 
-const WorkerPoolError = Entity.configure({
-  version: 1,
-  partitionKey: Entity.keys.StringKey('workerPoolId'),
-  rowKey: Entity.keys.StringKey('errorId'),
-  properties: {
-    // The worker pool this maps to.
-    workerPoolId: Entity.types.String,
-
-    // An arbitrary id for this error
-    errorId: Entity.types.SlugId,
-
-    // The datetime this error occured
-    reported: Entity.types.Date,
-
-    // The sort of error this is. Can be used by UIs to differentiate
-    kind: Entity.types.String,
-
-    // A human readable name for this error
-    title: Entity.types.String,
-
-    // A human readable description of this error and what can be done to fix it
-    description: Entity.types.String,
-
-    // Anything else that a reporter may want to add in a structured way
-    extra: Entity.types.JSON,
-  },
-});
-
-WorkerPoolError.prototype.serializable = function() {
-  return {
-    workerPoolId: this.workerPoolId,
-    errorId: this.errorId,
-    reported: this.reported.toJSON(),
-    kind: this.kind,
-    title: this.title,
-    description: this.description,
-    extra: this.extra,
-  };
-};
+class WorkerPoolError {
```

Comment:
I would double check with @imbstack but I suspect we wouldn't want to have a WorkerPoolError class but rather just access the db directly. https://github.com/taskcluster/taskcluster/pull/3105#discussion_r445239721

Reply:
@imbstack @djmitche Can you confirm?

Reply:
I only did a 5-minute scan but it looks like we pretty much only create/read these so yeah I think we can get away with just the db directly. This is not a strongly held position however!

Reply:
It would take some time to rewrite to have it not rely on the class. Given the rest of the tables of this service use a class to handle interactions with the db, maybe it's fine to have it reflect the same format for consistency. For most remaining services it will probably make sense to avoid having a `data.js` file.

Reply:
Either option sounds reasonable, or even keeping what's written but making an issue to switch it later.

Reply:
I left in the class wrapper in to match the rest of worker-manager - I can create an issue to change that if we think that's the right approach.